### DISCUSSION TITLE
Use Camera HNDL # for altered naming

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#548] [Feature Requests]
In multi-cam shoots, we can set HNDL=1-31

If we can automate this parameter in the altered file naming, I think it would be great.

Example:
H[ID]\_[yyddmm]\_

This would equate to something like:

H02_230919_GX123456.MP4

**Comment by dnewman-gpsw** [comment]
You can manually set you camera up that way (separate QR Codes.)  The Handle feature was added for a multicam stop-motion project, but I never heard of anyone as using it.  So doubt there are many users needing more extensions to the handle feature (does it even fully work today?)

**Comment by RikshaDriver** [comment]
I've set up the naming manually between the H11, 11 Mini & 12 for some multi-cam video tests at the moment.

It would make things a bit easier without having to clone multiple QR codes
